common/h/registers/riscv64_regs.h

```diff
@@ -0,0 +1,264 @@
+  DEF_REGISTER(       f22,  22 | FPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(       f23,  23 | FPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(       f24,  24 | FPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(       f25,  25 | FPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(       f26,  26 | FPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(       f27,  27 | FPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(       f28,  28 | FPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(       f29,  29 | FPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(       f30,  30 | FPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(       f31,  31 | FPR | Arch_riscv64, "riscv64");
+
+  // Aliases (ABI Name)
+  DEF_REGISTER(      zero,   0 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(        ra,   1 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(        sp,   2 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(        gp,   3 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(        tp,   4 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(        t0,   5 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(        t1,   6 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(        t2,   7 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(        fp,   8 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(        s0,   8 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(        s1,   9 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(        a0,  10 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(        a1,  11 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(        a2,  12 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(        a3,  13 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(        a4,  14 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(        a5,  15 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(        a6,  16 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(        a7,  17 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(        s2,  18 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(        s3,  19 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(        s4,  20 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(        s5,  21 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(        s6,  22 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(        s7,  23 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(        s8,  24 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(        s9,  25 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(       s10,  26 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(       s11,  27 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(        t3,  28 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(        t4,  29 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(        t5,  30 | GPR | Arch_riscv64, "riscv64");
+  DEF_REGISTER(        t6,  31 | GPR | Arch_riscv64, "riscv64");
+
+  // 32 bit FPRs f<N>_32 (Mainly for Capstone compatibility)
+  DEF_REGISTER(     f0_32,   0 | FPR | Arch_riscv64, "riscv64");
```

Comment:
`f<N>_32` and `f<N>_64` should be declared with DEF_REGISTER_ALIAS to make them explicit aliases.